.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: c
+sudo: false
+services:
+  - docker
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-docker.sh
+script: bash -ex ./.travis-docker.sh
+env:
+  global:
+  - PACKAGE="xenstore_transport"
+  - PINS="xenstore_transport:. xenstore-tool:."
```

Comment:
I copied from https://github.com/mirage/mirage-profile/blob/master/.travis.yml -- it's becoming really common over in mirage/ to define multiple top-level opam packages instead of findlib subpackages, so we have better control over dependencies (including depexts). I was pleased when I discovered that travis could be configured to test everything sensibly!